README.md

```diff
@@ -49,8 +49,9 @@ Follow these steps to add the library to your Android project:
            }
    ```
 
+For more details and guidance regarding configuration and requirements, see the [JWP Android SDK documentation](https://docs.jwplayer.com/players/docs/android-add-the-sdk).
```

Comment:
I think the Android link should be for the requirements heading like you did with iOS. 
```suggestion
For more details and guidance regarding configuration and requirements, see the [JWP Android SDK documentation](https://docs.jwplayer.com/players/docs/android-overview#requirements.
```